jenkins-editor-plugin/src/main/java-eclipse/de/jcup/jenkinseditor/preferences/JenkinsEditorPreferencePage.java

```diff
@@ -71,462 +71,487 @@
-        matchingBrackets = getBoolean(P_EDITOR_MATCHING_BRACKETS_ENABLED);
-        highlightBracketAtCaretLocation = getBoolean(P_EDITOR_HIGHLIGHT_BRACKET_AT_CARET_LOCATION);
-        enclosingBrackets = getBoolean(P_EDITOR_ENCLOSING_BRACKETS);
-
-        updateBracketUI();
-    }
-
-    private void reloadBracketHighlightingPreferenceDefaults() {
-        matchingBrackets = getDefaultBoolean(P_EDITOR_MATCHING_BRACKETS_ENABLED);
-        highlightBracketAtCaretLocation = getDefaultBoolean(P_EDITOR_HIGHLIGHT_BRACKET_AT_CARET_LOCATION);
-        enclosingBrackets = getDefaultBoolean(P_EDITOR_ENCLOSING_BRACKETS);
-
-        updateBracketUI();
-    }
-
-    private void updateBracketUI() {
-        this.bracketHighlightingCheckbox.setSelection(matchingBrackets);
-
-        this.enclosingBracketsRadioButton.setSelection(enclosingBrackets);
-        if (!(enclosingBrackets)) {
-            this.matchingBracketRadioButton.setSelection(!(highlightBracketAtCaretLocation));
-            this.matchingBracketAndCaretLocationRadioButton.setSelection(highlightBracketAtCaretLocation);
-        }
-        updateSlaveComponents();
-    }
-
-    private void updateSlaveComponents() {
-        for (MasterButtonSlaveSelectionListener listener : masterSlaveListeners) {
-            listener.updateSlaveComponent();
-        }
-    }
-
-    private class MasterButtonSlaveSelectionListener implements SelectionListener {
-        private Button master;
-        private Control slave;
-
-        public MasterButtonSlaveSelectionListener(Button master, Control slave) {
-            this.master = master;
-            this.slave = slave;
-        }
-
-        @Override
-        public void widgetDefaultSelected(SelectionEvent e) {
-
-        }
-
-        @Override
-        public void widgetSelected(SelectionEvent e) {
-            updateSlaveComponent();
-        }
-
-        private void updateSlaveComponent() {
-            boolean state = master.getSelection();
-            slave.setEnabled(state);
-        }
-
-    }
+		ComboFieldEditor comboFieldEditor = new ComboFieldEditor(name, labelText, entryNamesAndValues,
+				jenkinsCLIComposite);
+		addField(comboFieldEditor);
+
+		jarFileLocation = new FileFieldEditor(P_PATH_TO_JENKINS_CLI_JAR.getId(), "Path to jenkins-cli.jar (optional)",
+				jenkinsCLIComposite);
+		jarFileLocation.setFileExtensions(new String[] { "*.jar" });
+		jarFileLocation.getLabelControl(jenkinsCLIComposite).setToolTipText(
+				"You can set here the location of another jenkins-cli.jar which you can download by your running Jenkins instance.");
+		addField(jarFileLocation);
+
+		Label passwordLabel = new Label(jenkinsCLIComposite, SWT.LEFT);
+		passwordLabel.setText("User credentials : ");
+
+		Text passwordField = new Text(jenkinsCLIComposite, SWT.SINGLE | SWT.PASSWORD);
+		GridData data = new GridData(GridData.FILL_HORIZONTAL);
+		passwordField.setLayoutData(data);
+		if (temporaryCredentials == null) {
```

Comment:
Thank you for the changes.

It was a little bit hard to find the last changed parts, because of the applied auto formatter - there were so many whitespace changes... But maybe I should provide spotless or define some dedicated eclipse settings in future. 

But anyway, that's just whitespace stuff and the changes are all working. So I will approve the changes.